binnacle/README.md

```diff
@@ -19,6 +18,22 @@ A Flutter project.
 
 `flutter doctor` - Check if Flutter is installed properly
 
+`flutter clean` - Cleans build cache
+
+## Setting up
+
+### Google Maps API Key
+
+Get a Google Maps API key. Set the environment variable `OST_MAPS_API_KEY` to it.
+
+#### Windows
+
+Go to environment variables and add it.
+
+#### MacOS / Linux
+
+In your bash config add `OST_MAPS_API_KEY=YOUR_KEY_HERE` and reload the config.
```

Comment:
That way doesn't work. As far as I can tell the only (and best) way to do this is with an untracked file.